website/pages/en/chain-integration-overview.mdx

```diff
@@ -6,12 +6,12 @@ A transparent and governance-based integration process was designed for blockcha
 
 ## Stage 1. Technical Integration
 
-- Teams work on a Graph Node integration and Firehose for non-EVM based chains. [Here's how](/new-chain-integration/).
+- Please visit [New Chain Integration](https://thegraph.com/docs/en/new-chain-integration/) for information on graph-node support for new chains.
```

Comment:
```suggestion
- Please visit [New Chain Integration](/new-chain-integration) for information on graph-node support for new chains.
```
We use locale-relative URLs as much as possible, since the locale should change in translated versions of the page but not what follows.